### ISSUE TITLE
Qemu warning on OS X

### ISSUE THREAD

**Issue by GhostLyrics** [IAIK/sweb#56]
When building a `qemu` target on OS X, the following warning appears:

```none
| ghostlyrics@Razgriz: /tmp/build
| => make qemu
Scanning dependencies of target qemu
Executing `qemu-system-i386 -m 8M -cpu qemu32 -hda SWEB-flat.vmdk -debugcon stdio -no-reboot`
WARNING: Image format was not specified for 'SWEB-flat.vmdk' and probing guessed raw.
         Automatically detecting the format is dangerous for raw images, write operations on block 0 will be restricted.
         Specify the 'raw' format explicitly to remove the restrictions
```

I am considering rewriting those lines in all CMakeLists.include(s) to be something along the lines of the solution in [a similar issue from grub2-theme-preview](https://github.com/hartwork/grub2-theme-preview/commit/0cb285d98adeb644c50a915d6f350a56daa0cba2?diff=split).

```diff
-COMMAND	qemu-system-i386 -m 8M -cpu qemu32 -hda SWEB-flat.vmdk -debugcon stdio -no-reboot
+COMMAND	qemu-system-i386 -m 8M -cpu qemu32 -drive file=SWEB-flat.vmdk,index=0,media=disk,format=raw -debugcon stdio -no-reboot
```

I've tested this for the x86/32 version and it boots up fine. Thoughts?

**Comment by GhostLyrics** [comment]
Also tested this in the VM for x86/32. Will run all builds and get back.
Do you want me to leave the old version in the TACOS targets for now?

**Comment by GhostLyrics** [comment]
Should be fixed by #59

**Comment by dgruss** [comment]
merged #59
